client/network/src/service.rs

```diff
@@ -1025,6 +1043,27 @@ impl<B: BlockT + 'static, H: ExHashT> NetworkService<B, H> {
 			.to_worker
 			.unbounded_send(ServiceToWorkerMsg::OwnBlockImported(hash, number));
 	}
+
+	// Utility function to extract `PeerId` from each `Multiaddr` for priority group updates.
```

Comment:
```suggestion
	/// Utility function to extract `PeerId` from each `Multiaddr` for priority group updates.
	///
	/// Returns an `Err` if one of the given addresses is invalid or contains an
	/// invalid peer ID (which includes the local peer ID).
```

Just to make sure this is not used somewhere where the local peer ID is valid.